.nvmrc

```diff
@@ -0,0 +1 @@
+v20.14.0
```

Comment:
adding `.nvmrc` files is something I've just gotten into the habit of doing, it's a nice quality of life developer experience addition